### PR TITLE
fix(app): Add the version and show diddocument in the dev mode

### PR DESCRIPTION
**Pull request by talwinder50**
IMG_0537

Signed-off-by: Talwinder kaur <talwinder.kaur@avast.com>

### CODE REVIEW

File: demo/app/ios/Runner/flutterPlugin.swift [hunk 1]
```diff
@@ -67,6 +67,9 @@ public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
 
         case "activityLogger":
             storeActivityLogger(result:result)
+        
+        case "getVersionDetail":
```
Comment (vkubiv) [inline demo/app/ios/Runner/flutterPlugin.swift:71]: maybe rename it to getWalletSDKVersion or something like this, to not confuse it with the app version

Comment (talwinder50) [reply demo/app/ios/Runner/flutterPlugin.swift:71]: yea make sense will do that

File: demo/app/ios/Runner/flutterPlugin.swift [hunk 2]
```diff
@@ -89,7 +92,16 @@ public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
...
-    
+     
+    /**
+              This method gets the version detail if we build sdk using the env variable
+              For Example: NEW_VERSION=testVer GIT_REV=testRev BUILD_TIME=testTime make generate-ios-bindings copy-ios-bindings
+     */
+    public func getWalletSDKVersion(result: @escaping FlutterResult) {
+        let version = VersionGetVersion()
+        print("getting version detail ->", version)
+        result(version)
```
Comment (rolsonquadras) [inline demo/app/ios/Runner/flutterPlugin.swift:103]: Can we show Gitversion as well as Build time as well ?

https://github.com/trustbloc/wallet-sdk/blob/main/demo/app/ios/Runner/Tests/IntegrationTest.swift#L25-L26

Comment (talwinder50) [reply demo/app/ios/Runner/flutterPlugin.swift:103]: Yes can add that too. Will add n update

**Review by rolsonquadras — APPROVED** [review]

**Review by DRK3 — APPROVED** [review]
Looks good, would suggest renaming `getVersionDetails` as @birtony suggested before merging

**Pull request merged**